.github/workflows/ci.yml

```diff
@@ -1727,24 +1734,6 @@ jobs:
         with:
           persist-credentials: false
           submodules: recursive
-      - name: "Install Breeze"
```

Comment:
It's also faster as we do not need to install breeze/pull image since we are using constraints generated earlier and download them from artifacts